gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,9 +1,8 @@
-#Tue Feb 22 14:34:43 IST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
```

Comment:
`bin` zip is cross platform?

Reply:
The all variant has extra docs/samples included this change was automatically done with gradlew update command